test/protected_resource_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'securerandom'
```

Comment:
This works fine when run as `bundle exec rake test`, but did not work in isolation and uses `SecureRandom`.